examples/siro_sandbox/sandbox_app.py

```diff
@@ -1040,6 +1040,15 @@ def _sim_update_tutorial(self, dt: float):
         else:
             self.cam_transform = self._tutorial.get_look_at_matrix()
 
+    def _add_spatial_data_to_gfx_replay_keyframe(self):
+        sim = self.get_sim()
+        agent_node = sim.agents_mgr[0].articulated_agent.sim_obj
```

Comment:
For some reason if I record the agents pose like it is done in the sim [gfx replay tutorial](https://github.com/facebookresearch/habitat-sim/blob/03760214d1f416bb2361b3a57d6ea1ac7e9941b3/examples/tutorials/nb_python/replay_tutorial.py#L113)
```
 agent_node = sim.get_agent(0).body.object
 gfx_replay_utils.add_node_user_transform(sim, agent_node, "agent")
```
agent_node position/orientation is not updated as the agent moves in the scene. So, in this draft PR I do it like this.